pages/common/kops.md

```diff
@@ -0,0 +1,24 @@
+# kops
+
+> A tool that create, destroy, upgrade and maintain production-grade, highly available, Kubernetes clusters from the command line.
+> More informations: <https://github.com/kubernetes/kops/>.
+
+- Create a cluster from the configuration specification:
+
+`kops create cluster -f {{my-cluster.yaml}}`
+
+- Create secret from secret spec file:
+
+`kops create -f {{secret.yaml}}`
+
+- Export configurations from a cluster:
+
+`kops export kubecfg {{my-cluster}}`
```

Comment:
```suggestion
`kops export kubecfg {{cluster_name}}`
```
Could we have all these tokens use an underscore instead of a dash? Just to be consistent with other pages. Also, let's make it impersonal `cluster_name` vs `my_cluster`.

Same applies to every token in the page.

Reply:
👍 makes sense